src/docs/organization.yaml

```diff
@@ -218,7 +218,54 @@ paths:
                                     message:
                                         type: string
                                         example: 'Invalid Token: No authorization token was found'
-
+    /api/organization/{id}/sub:
+        post:
+            tags:
+                - Organization
+            summary: Delete organization sub
+            description: Delete organization sub
+            operationId: unFollowOrga
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            parameters:
+                - in: path
+                  name: id
+                  description: The id of the organization
+                  required: true
+                  type: string
+            responses:
+                404:
+                    description: not found values
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    error:
+                                        type: string
+                                        example: Organization not found
+                200:
+                    description: Organization Unsubscribed
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    message:
+                                        type: string
+                                        example: Unfollowed Successfully!                
+                422:
```

Comment:
@muhammed-shihebi do we really need this error here? since we don't have any value added in the request?

Reply:
Sorry why would you mention shehabi in my code! LOL : D

Reply:
Jokking! i think since we used **VERIFY_VALIDATION_FUNDSBYID,** in the route then i may get this error one day : D